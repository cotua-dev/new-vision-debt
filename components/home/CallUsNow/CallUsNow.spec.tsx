import { render, RenderResult } from '@testing-library/react';
import { CallUsNow } from './index';

describe('CallUsNow Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<CallUsNow/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
