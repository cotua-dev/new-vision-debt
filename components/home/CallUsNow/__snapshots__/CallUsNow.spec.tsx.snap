// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CallUsNow Component renders 1`] = `
<section
  class="debt-advisors-call-us-now"
>
  <div
    class="content-wrapper"
  >
    <strong
      class="content-text"
    >
      For expert debt resolution advice you can trust the licensed partners of New Vision Debt
    </strong>
    <a
      class="phone-link"
      href="tel:+18009211420"
    >
      <span
        class="number"
      >
        800-921-1420
      </span>
      <span
        class="text"
      >
        CALL US NOW
      </span>
    </a>
  </div>
</section>
`;
